### ISSUE TITLE
You stole

### ISSUE THREAD

**Issue by jasonczc** [keithkfng/HungerGames#3]

**Comment by jasonczc** [comment]
https://github.com/fengberd/FSurvivalGame
You STOLE IT.

**Comment by keithkfng** [comment]
LOL this is a fork stupid. Can't you use your eyes before #blame?

**Comment by UltimateMcraft** [comment]
LOL this is a fork, and the original project is open-source, anyone can contribute, at the top have a look, it says FORKED

**Comment by NguyenQuangLap** [comment]
haha

**Comment by GlitchPlayer** [comment]
hahaha, what a shame.

**Comment by applqpak** [comment]
Lol. xD

**Comment by keithkfng** [comment]
Sometimes people just need to read with their eyes before typing with their fingers XD

**Comment by GlitchPlayer** [comment]
Maybe he's blind XD

**Comment by keithkfng** [comment]
True, but he wouldn't be able to establish and search for the repository I had "stolen" from if he was blind.

**Comment by applqpak** [comment]
Lol.
